src/server/ua_subscription_monitoreditem.c

```diff
@@ -121,6 +121,8 @@ UA_Notification_dequeue(UA_Server *server, UA_Notification *n) {
     TAILQ_REMOVE(&mon->queue, n, listEntry);
     --mon->queueSize;
 
+    mon->eventOverflows = 0;// see open issue #2114
```

Comment:
Better add a more detailed comment here, so that one does not need to open the issue to understand why this is necessary